src/core/core.ts

```diff
@@ -25,8 +25,8 @@ export class Core {
   /**
    * Creates document for this group in Cloud Firestore.
    */
-  async initNewGroup(groupId): Promise<void> {
-    await admin.firestore().collection("groups").doc(groupId).create({
+  initNewGroup(groupId): void {
```

Comment:
It performs an async operation, so let's make it clear. I'd leave this as it was before.

Reply:
a bit useless but ok

Reply:
done

Reply:
not useless
if you don't want to wait, you can just do `initNewGroup(groupId)` without awaiting it
and it's more explicit, doesn't return until it's done